common/models/acl.js

```diff
@@ -515,8 +518,11 @@ module.exports = function(ACL) {
    * @param {Function} cb Callback function
    */
   ACL.resolvePrincipal = function(type, id, cb) {
```

Comment:
yes, let's land it in the other PR, it's better to keep things separated, although https://github.com/strongloop/loopback/pull/ is currently dealing only with Role, not ACL